crocks/Pred.js

```diff
@@ -37,7 +40,7 @@ function Pred(runWith) {
       throw new TypeError('Pred.concat: Pred required')
     }
 
-    return Pred(x => runWith(x) && m.runWith(x))
+    return Pred(x => !!runWith(x) && !!m.runWith(x))
```

Comment:
This is ready for another go around, please let me know if you feel good about the coverage around this change. Do not want this getting introduced again